client/src/components/App.jsx

```diff
@@ -66,6 +67,20 @@ class App extends React.Component {
       });
   }
 
+  updateLike() {
+    alert("I was clicked")
+    let boolNum = this.state.is_liked;
+    let id = this.state.id;
+    if( boolNum === 0) {
+      this.setState({ is_liked : 1});
+    } else {
+      this.setState({ is_liked : 0});
+    }
+    // var params = [id, boolNum]
+    // axios.get('/updateLike')
+componentDidUpdate
```

Comment:
i cant tell what componentDidUpdate is doing here

Reply:
experiment here. for using the modal